src/box/lua/connpool.lua

```diff
@@ -71,24 +78,49 @@ local function connect(instance_name, opts)
     return conn
 end
 
+local function is_candidate_connected(candidate)
+    local conn = connections[candidate]
+    return conn and conn.state == 'active' and conn:mode() ~= nil
+end
+
+-- Checks whether the candidate instance is not in some of the
+-- connecting/fetching state.
```

Comment:
nit: for me this comment was not as clear as the function name and body itself.

Reply:
Fixed the comment to a bit more clearer (I suppose). Thanks.